.ddev/docker-compose.tempfs.yaml

```diff
@@ -0,0 +1,9 @@
+version: '3.6'
+
+services:
+  web:
+    volumes:
+      - type: tmpfs
+        target: /var/www/html/.Build/var/cache
+        tmpfs:
+          size: 268435456
```

Comment:
just out of interest: how much does this improve the performance?

Reply:
I don't know but my guess this also reduces file locking issues see ddev Slack channel. Perhaps @rfay can say more about this...

Reply:
I haven't tested performance, but it resolves docker mount problems on Windows quite a lot. There's a regularly recurring docker/TYPO3 problem (https://stackoverflow.com/questions/57554778/how-can-i-fix-the-caching-issue-in-typo3-and-ddev/58317553#58317553) that is usually resolved bby either tmpfs mounting or using NFS instead of regular docker mounting. I haven't actually heard that it improves perceived performance, but it does make happier Windows users. In reality, TYPO3 should probably just symlink /var to /tmp and it would be a better solution.

Reply:
Not exactly since TYPO3's `var` does not only contain caches but also persistent data like logs, localizations, etc. That's not something you'd expect to be gone after a restart.

Reply:
Never had issues because this files are deleted on a restart but also never had mounting problems because of using this. And it's only set for var/cache, so no logs etc should be deleted at all...